IronMan_MK1/tests/data/parse_question.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+import json
+import re
+
+def writeJosn(filename,data):
+    file_full_name = './'+filename+'.json'
+    with open(file_full_name,'w') as file_out:
+        json.dump(data, file_out,indent=4)
+
+
```

Comment:
- the code below should be inside an if __name__ == "__main__" block

Reply:
fixed